packages/apm/src/hubextensions.ts

```diff
@@ -57,7 +55,7 @@ function startSpan(spanOrTransactionContext: SpanContext | TransactionContext):
     // We only roll the dice on sampling for root spans of transactions because all child spans inherit this state
     if (transaction.sampled === undefined) {
       const sampleRate = (client && client.getOptions().tracesSampleRate) || 0;
-      transaction.sampled = Math.random() < sampleRate;
+      transaction.sampled = Math.random() <= sampleRate;
```

Comment:
It should stay `<`, I removed my previous comment and thought you won't see it 😅.
`Math.random` is `0` inclusive, `1` exclusive so for default of `0`:

```
transaction.sampled = 0.01 <= 0; // false - drop event
transaction.sampled = 0 <= 0; // true - wont drop event but should
```